.github/workflows/acceptance.yml

```diff
@@ -72,7 +72,7 @@ jobs:
 
   acceptance:
     # TODO(joshuarli): Convert to py3 with snapshots. See other TODO as well.
-    name: python2.7 acceptance
+    name: python acceptance
```

Comment:
It doesn't affect Freight because this is on `sentry` repo, it will affect GitHub settings (merging PRs)
![image](https://user-images.githubusercontent.com/79684/105078523-fdf58700-5a42-11eb-9ad5-a1b97fdea616.png)